src/index.js

```diff
@@ -1,101 +1,129 @@
+import DoctorView from "./routes/doctorview";
+import PatientInsurance from "./routes/patientinsurance";
 import InsurancePlans from "./routes/providerinsuranceplans";
-
-
+import DoctorDashboard from "./routes/doctordashboard";
+import PatientDashboard from "./routes/patientdashboard";
+import ChatApp from "./DashboardPages/ChatApp";
+import PatientDetails from "./DashboardPages/PatientDetails";
+import PatientDetailsInfo from "./DashboardPages/PatientDetailsInfo";
 const router = createBrowserRouter([
   {
     path: "/",
     element: <Root />,
   },
   {
-    path:"registration",
+    path: "registration",
     element: <Registration />,
   },
   {
-    path:"registration/enroll",
-    element: <DuoEnroll />
+    path: "registration/enroll",
+    element: <DuoEnroll />,
   },
   {
-    path:"login",
-    element: <Login />
+    path: "login",
+    element: <Login />,
   },
   {
-    path:"login/auth",
-    element: <DuoAuth />
+    path: "login/auth",
+    element: <DuoAuth />,
   },
   {
-    path:"insurance",
-    element: <InsuranceHome />
-  }, 
+    path: "insurance",
+    element: <InsuranceHome />,
+  },
   {
-    path:"search",
-    element: <Search />
+    path: "search",
+    element: <Search />,
   },
   {
-    path:"appointment",
-    element: <Appointment />
-  }, 
+    path: "appointment",
+    element: <Appointment />,
+  },
+
   {
-    path:"doctor",
-    element: <DoctorHome />
+    path: "patient",
+    element: <PatientHome />,
   },
   {
-    path:"patient",
-    element: <PatientHome />
+    path: "patient/profile",
+    element: <PatientProfile />,
   },
   {
-    path:'patient/profile',
-    element: <PatientProfile />
+    path: "patient/doctorview",
+    element: <DoctorView />,
   },
   {
-    path:'doctor/profile',
-    element: <DoctorProfile />
+    path: "patient/patientchat",
+    element: <ChatApp />,
   },
   {
-    path:'insurance/profile',
-    element: <InsuranceProfile />
+    path: "patient/insurance",
+    element: <PatientInsurance />,
   },
   {
-    path:'patient/doctorview',
-    element: <DoctorView />
+    path: "doctor",
+    element: <DoctorHome />,
   },
   {
-    path:'patient/insurance',
-    element: <PatientInsurance />
+    path: "doctor/profile",
+    element: <DoctorProfile />,
   },
   {
-    path:'insurance/plans',
-    element: <InsurancePlans />
-  }
+    path: "doctor/doctorchat",
+    element: <ChatApp />,
+  },
+  {
+    path: "doctor/patientdetails/*",
+    element: <PatientDetails />,
+  },
+  {
+    path: "doctor/patientdetails/patientdetailsInfo/:patientID/:doctorID",
+    element: <PatientDetailsInfo />,
+  },
+
+  {
+    path: "insurance/doctorchat",
+    element: <InsuranceProfile />,
+  },
+  {
+    path: "insurance/plans",
+    element: <InsurancePlans />,
+  },
+  {
+    path: "doctordashboard/*",
+    element: <DoctorDashboard />,
+  },
+  {
+    path: "patientdashboard/*",
+    element: <PatientDashboard />,
+  },
```

Comment:
Are the doctordashboard and patientdashboard routes still in use? If they're not they should prob be removed.